workshops/test/test_util.py

```diff
@@ -0,0 +1,138 @@
+# this probably needs refactored for py3
+from datetime import datetime
+from StringIO import StringIO
+from json import dumps
+
+from django.test import TestCase
+
+from ..models import Site, Event, Role, Person
+from ..util import PERSON_UPLOAD_FIELDS, upload_person_task_csv,\
+    verify_upload_person_task
+
+from .base import TestBase
+
+class UploadPersonTaskCSVTestCase(TestCase):
+
+    def compute_from_string(self, csv_str):
+        ''' wrap up buffering the raw string & parsing '''
+        csv_buf = StringIO(csv_str)
+        # compute & return
+        return upload_person_task_csv(csv_buf)
+
+    def test_basic_parsing(self):
+        ''' See views.PERSON_UPLOAD_FIELDS for field ordering '''
+        csv = """personal,middle,family,email\njohn,a,doe,johndoe@email.com\n,jane,a,doe,janedoe@email.com"""
```

Comment:
(Personal preference) I'd rather have this split into multiple lines and not use `"\n"` explicitely. You have `"""` already.